ONSdigital/eq-questionnaire-runner
# Bug Fix: Decimals are only showing first 3 decimal places

### What is the context of this PR?

Numbers with more than 3 decimal places are being rounded to only show the first 3 places on summary pages and in placeholders.

Have changed formatted_number and format_unit to ensure that rounding does not take place and where necessary, all decimal places are displayed.

### How to test/review
Run ```make test-unit``` to ensure they pass. Test any schemas that accept decimals and ensure that there is no rounding taking place. You may need to edit "decimal_places:" within the schema files to allow for extra decimals to be entered e.g below:

Take any schema that shows answers at the end and with number/unit questions, such as test_unit_patterns.json, and edit one of the answer fields to have

```"decimal_places": 6```

enter a value for the field with 6 decimal places and continue to the survey summary.

It should show all 6 decimal places on the summary and anywhere else the answer is used if the max has been set at 6 (should always show the number of decimal places the user has entered).
